pkgs/top-level/all-packages.nix

```diff
@@ -15664,6 +15666,8 @@ with pkgs;
     inherit (gnome3) gtksourceview;
   };
 
+  dronin = callPackage ../applications/science/robotics/dronin { };
```

Comment:
This will need to be fixed, see c327df544ae0875ce71b0a82ae7a650a8e34a268 and aa4707fc3449192b961bfd2dfc16b9042df0d7f3